cmds.pl

```diff
@@ -21,6 +21,8 @@ sub task_add {
     &draw_prompt_line('');
     return;
   }
+  # TODO: fix this evilness
```

Comment:
From what I understand, escaping is all we can do, because we cannot quote the string because we want taskwarrior to parse it (e.g. the string might be "buy a cup of coffee +caffeine project:life").

Your comment suggests there's a more robust approach, but I don't see it. What do you suggest for the long-run?